pwa/src/components/applications/applicationForm.tsx

```diff
@@ -51,6 +53,7 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({id}) => {
         setApplication(res.data)
       })
       .catch((err) => {
+        setAlert({message: err, type: 'danger'})
```

Comment:
Works like a charm!

I think for the error messages, we want to send two pieces of data to the user:

1. An generic message like "Oops something went wrong" and
2. The actual error message.

Right now, we're only pushing the error message to the user. Could you create a JIRA issue for us to update this error Alert in the future?